apache-spark-worker/build/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM ubuntu
+
+MAINTAINER Frank Austin Nothaft, fnothaft@berkeley.edu
+
+RUN apt-get update && \
+  apt-get install -y \
+    python \
+    libnss3 \
+    openjdk-8-jdk \
+    curl
+
+COPY download.sh /home/
+RUN sh /home/download.sh
+
+WORKDIR /opt/apache-spark
+RUN sed -i "s/<joda.version>2.5<\/joda.version>/<joda.version>2.9.4<\/joda.version>/" pom.xml
```

Comment:
I agree. I can also (instead of or in addition to using patch) use xmlstarlet (http://xmlstar.sourceforge.net/) instead of sed if you prefer though that means installing one more thing in the container.